### PR TITLE
add MIT E-Vent - MIT Emergency Ventilator

### PR DESCRIPTION
**Pull request by branciard**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,4 +70,13 @@ An open hardware and source project aiming to upgrade ventilation bags. By colla
   <img width="400px" src="https://github.com/bneiluj/covid-19-open-source-ventilators/blob/master/assets/CORESPONSE_2.jpg" />
 </p>
 
+
+**MIT E-Vent - MIT Emergency Ventilator**
+
+- Website: https://e-vent.mit.edu/
+- Twitter Post David Austin from MIT: https://twitter.com/sinanaral/status/1244107247959056384?s=20
+- Media Post : https://scitechdaily.com/mit-posts-free-plans-online-for-an-emergency-ventilator-that-can-be-built-for-100/
+
```
Comment (bneiluj) [inline README.md]: Can you please add an image of the project ?

Comment (branciard) [reply README.md]: image added

**Comment by bneiluj** [comment]
Reward: 5 DAI

@branciard : please post your ethereum address here.

As explained here: https://gitcoin.co/grants/578/open-source-covid-ventilator-oscv?tab=description)

**Comment by branciard** [comment]
Thank you @bneiluj . Here an ETH address for reward : 0x904b23304AffC3B2d2b7c577708055D7C8eCf63A

**Comment by bneiluj** [comment]
Sure thing. When the gitcoin stage is finished. 

Is there any github attached to MIT Emergency Ventilator ?

**Comment by branciard** [comment]
I do not find github, I see only details on website. maybe more on resources section, but register is needed : https://e-vent.mit.edu/resources/downloads/

**Comment by bneiluj** [comment]
should we add files here? Is it legally possible ?

**Pull request merged**